app/code/Magento/Theme/view/frontend/templates/html/topmenu/renderer.phtml

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
It's just a remark to whoever is going to be reviewing the pull request. I'm not involved into the PR acceptance process :)